functions/Get-DbaLogin.ps1

```diff
@@ -0,0 +1,135 @@
+Function Get-DbaLogin
+{
+<#
+.SYNOPSIS 
+Function to get an SMO login object of the logins for a given SQL Instance. Takes a server object from the pipe 
+
+.DESCRIPTION
+The Get-DbaLogin function returns an SMO Login object for the users passed, if there are no users passed it will return all logins.  
+
+.PARAMETER SqlInstance
+The SQL Server instance.You must have sysadmin access and server version must be SQL Server version 2000 or higher.
+
+.PARAMETER SqlCredential
+Allows you to login to servers using SQL Logins as opposed to Windows Auth/Integrated/Trusted. 
+
+.PARAMETER Logins 
+Pass a single login, or a list of them. Comma delimited. 
+
+.PARAMETER NoSystemLogins
+A switch to exlude system logins from returning. (Logins with '##' in the front or the 'sa' login)
```

Comment:
true,

if we use -NoSystem... it should really mean the systemobjects. So the switch can stay, but the statement should be `Where -not IsSystemObject` instead of the ## and 'sa'

we probably better leave all other excludes to the user with a 
`Get-DbaLogin | where ...`